theme/themes/pastanaga/elements/label.overrides

```diff
@@ -6,3 +6,17 @@
     margin: 0em 0em 0em 0.75em;
   }
 }
+
+.ui.form {
+  .ui.pointing.label:before {
+    content: initial;
+  }
```

Comment:
@sneridagh I think we should remove `pointing` from https://github.com/plone/volto/blob/master/src/components/manage/Widgets/FormFieldWrapper.jsx#L99 instead of changing the styles for `.pointing`. Nothing is pointing at anything now.

We could also add a new class in FormFieldWrapper and apply our styles to that class. I think that would help avoid breaking changes because if someone has shadowed FormFieldWrapper.jsx, the appearance won't change.